docs/reference/auto-updates.md

```diff
@@ -0,0 +1,14 @@
+---
+id: auto-updates
+title: "Auto-updates"
+description: "Some updates are applied to SaaS clusters automatically."
+---
+
+Camunda 8 SaaS customers can enable auto-updates. When enabled, the cluster will be updated once a new patch release is available.
```

Comment:
It would be great to mention where in Saas Menu this could be enabled.

Reply:
Where is this exactly? See my comment in the description

> @felix-mueller I would like to add a page to the Console > Manage clusters area, but I don't know how to enable auto-updates. Can you add the steps in a comment and I will add them to this PR?

Reply:
![Screenshot 2023-10-24 at 16 54 51](https://github.com/camunda/camunda-docs/assets/20283848/91700f22-6ea3-4796-af19-d8097f418bae)
![Screenshot 2023-10-24 at 16 54 40](https://github.com/camunda/camunda-docs/assets/20283848/e9abac5c-68b6-4287-b7ec-390d226797d5)

Reply:
Cluster Settings Page for existing clusters
For new it's part of the creation flow